Lib/test/test_dis.py

```diff
@@ -2483,6 +2483,11 @@ def f():
                     with self.subTest(args=args[1:]):
                         _ = self.invoke_dis(*args)
 
+        with self.assertRaises(SystemExit):
+            # suppress argparse error message
+            with contextlib.redirect_stderr(io.StringIO()):
```

Comment:
I think this works

```suggestion
            with contextlib.redirect_stderr(None):
```

Reply:
Actually there's another `redirect_stdout` inside `invoke_dis`. Do we need two?

Reply:
Here's it's to suppress the message by argparse (otherwise there is a "unknown argument" message in the test output). Note that it's supressing stderr and not stdout.

Reply:
Does `None` work?

Reply:
It appears that it works since it sets sys.stderr to None, but is it an expected behaviour?

Reply:
I think it is, stderr is allowed to be `None`.

Reply:
let's leave it as it is, it doesn't matter.